apps/frontend-v3/app/(app)/pools/cow/page.tsx

```diff
@@ -37,7 +37,7 @@ export default function PoolsPage() {
       <DefaultPageContainer noVerticalPadding pb={['xl', '2xl']} pt={['lg', '54px']}>
         <FadeInOnView animateOnce={false}>
           <Suspense fallback={<Skeleton h="500px" w="full" />}>
-            <PoolList fixedPoolTypes={[GqlPoolType.CowAmm]} hidePoolTags={['VE8020']} />
```

Comment:
can't do this anymore
alternative: create a separate app for cow -> cow-pools.balancer.fi

Reply:
Not a fan of having to create another app for this feature tbh.

Reply:
it would also be easier with theming and could remove swap & vebal from navbar?
another alternative is to hide the pool tag in the pool filter as other stuff is also already dependent on `isCowPath`